libraries/AP_UAVCAN/AP_UAVCAN.h

```diff
@@ -118,6 +118,8 @@ class AP_UAVCAN {
         float remaining_capacity_wh;
         float full_charge_capacity_wh;
         uint8_t status_flags;
+        uint32_t model_instance_id;
+        uint8_t model_name[32+1];
```

Comment:
Can we have more insight into what does '32' and '1' stand for? We could define them as a const maybe?